launcher/ui/pages/global/LauncherPage.cpp

```diff
@@ -184,6 +184,11 @@ void LauncherPage::on_downloadsDirBrowseBtn_clicked()
     }
 }
 
+void LauncherPage::on_downloadsDirWatchRecursiveCheckBox_clicked()
+{
+  // incase anything needs to be done here
```

Comment:
if something needs to be done here in the future, we can add the method, there's no need to add it if we don't use it now.